src/ml-service/packages/sensor_location/app/modules/locate_model.py

```diff
@@ -0,0 +1,52 @@
+import pandas as pd
+from pymongo import MongoClient
+import json
+
+def connect_mongo():
+    '''
+    Connects to local MongoDB
+    '''
+    #client = MongoClient('mongodb+srv://lillian:fosho@cluster0-99jha.gcp.mongodb.net/test?retryWrites=true&w=majority')
+    client = MongoClient("mongodb://localhost:27017")
```

Comment:
Noticed the switch to using localhost, could there be a challenge with ATLAS? The resource issue?